mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -1080,12 +1100,40 @@ pub async fn nexus_create(
         return Ok(());
     }
 
+    let (min_cntlid, max_cntlid, resv_key) =
+        nvme_params.unwrap_or((NVME_MIN_CNTLID, NVME_MAX_CNTLID, 0x1234_5678));
```

Comment:
in a struct we could return defaults.

Reply:
There is now a `impl Default for NexusNvmeParams`.